.devcontainer/requirements.txt

```diff
@@ -18,3 +18,5 @@ ruff==0.4.6
 textalloc==1.1.3
 shapely==2.0.6
```

Comment:
compared to a standard map, it's not visually clear why parks aren't drawn as well . Besides, The forests bring a lot of contrast and tend to hide the river, roads and track, which should be the most important elements. We could use a color closer to the background but in this case what's the point of collecting the data ?  :thinking: 

Like for the railways, could you keep all the optim and improvments but remove airport and forests ? :confused: 

You can keep them in a separate branch in case it makes sense for a future use

![image](https://github.com/user-attachments/assets/ce35daab-92b1-40a0-8c7a-3e7920ecec56)
![image](https://github.com/user-attachments/assets/08c48657-4329-4038-a8a7-992993caf8f7)

Reply:
Was thinking of adding an option to activate or not the railways and the forest and the airports with a checkbox.

For runs in smaller cities I believe that adding these details is a good option :)

Reply:
Do you have some examples for each category (railway, airport, forest) where it would make sense ?

Reply:
For railways and a bit of airport. This run is fine.
In fact, all small runs can support more details such as railways, airports or forests

![Drancy10km](https://github.com/user-attachments/assets/2f297c28-5d48-4fa1-a1f7-90c458597739)

Reply:
@ThomasParistech Thickness should also be scaled depending on a caracteristic length, I am going to add this in a next PR

Reply:
Here for the forest, I agree that the color is not perfect but I believe adding this data is cool.
![Route_4_chateaux](https://github.com/user-attachments/assets/aa8f3aa3-7e9d-497c-ae9e-fb3a7190309f)